### DISCUSSION TITLE
Missions - Add a section in icons to change the missions GUI when the user can't yet complete it

### DISCUSSION THREAD

**Discussion by banicola** [BG-Software-LLC/SuperiorSkyblock2#220] [Pending Feature Requests] [answered]
Currently you can only change not-completed, can-complete and completed but there's no "no-required-missions" that allows to display something different if the user didn't complete the missions in the required-missions section. 

That means that for the user, he sees the GUI with a message saying that he can progress into the mission but his progression doesn't show up since he can't yet do the mission.

**Comment by Lemvs** [comment, marked as answer]
Implemented some time ago as a 'locked' icon.

@OmerBenGera you can mark this answered and close it.
